### PR TITLE
WIP: Add trap into bash for LD_LIBRARY_PATH

### LINKED ISSUE #275: Broken game remediation through traps
Experiment with using forced initrc and traps to mitigate broken shell wrappers in games.
(eg ENV=/app/etc/shinitrc)

### PR DESCRIPTION
**Pull request by nanonyme**
Try to prevent games from removing Steam runtime from LD_LIBRARY_PATH. Closes #275

### CODE REVIEW

File: entrapment.sh [hunk 1]
```diff
@@ -0,0 +1,11 @@
+_OLD_LD_LIBRARY_PATH=$LD_LIBRARY_PATH
+function ld_library_path() {
+    if [ -n "$_OLD_LD_LIBRARY_PATH" -a "$_OLD_LD_LIBRARY_PATH" != "$LD_LIBRARY_PATH" ]
+    then
+	LD_LIBRARY_PATH=$LD_LIBRARY_PATH:$_OLD_LD_LIBRARY_PATH
```
Comment (TingPing) [inline entrapment.sh]: whitespace!

Comment (nanonyme) [reply entrapment.sh]: I really need to teach my Emacs to use spaces since I always manually use them anyway...

**Comment by nanonyme** [comment]
@TingPing any opinions on the order? If I set the order the other way around, it might potentially fix #95 but maybe break some other games.

**Comment by nanonyme** [comment]
Probably need to make the trap not be used for steam.sh at least

**Comment by nanonyme** [comment]
Looks like this won't work, closing

**Pull request closed without merging**